workflows/sarek.nf

```diff
@@ -267,6 +267,13 @@ include { BAM_VARIANT_CALLING_TUMOR_ONLY_ALL             } from '../subworkflows
 // Variant calling on tumor/normal pair
 include { BAM_VARIANT_CALLING_SOMATIC_ALL                } from '../subworkflows/local/bam_variant_calling_somatic_all/main'
 
+// Concatenation of germline vcf-files
+include { BCFTOOLS_CONCAT                                } from '../modules/nf-core/bcftools/concat/main'
+
+include { BCFTOOLS_SORT as GERMLINE_VCFS_CONCAT_SORT     } from '../modules/nf-core/bcftools/sort/main'
+
+include { TABIX_TABIX as TABIX_GERMLINE_VCFS_CONCAT_SORT } from '../modules/nf-core/tabix/tabix/main'
```

Comment:
```suggestion
include { BCFTOOLS_CONCAT                                                              } from '../modules/nf-core/bcftools/concat/main'
include { BCFTOOLS_SORT as GERMLINE_VCFS_CONCAT_SORT     } from '../modules/nf-core/bcftools/sort/main'
include { TABIX_TABIX as TABIX_GERMLINE_VCFS_CONCAT_SORT  } from '../modules/nf-core/tabix/tabix/main'
```

Reply:
Hi Friederike. I'd be happy to just commit your suggestion, but why the long space after `BCFTOOLS_CONCAT`?  As far as I can see, it just "breaks" the alignment.

Was your main point to just delete the blank lines?

Reply:
yes, actually it was showing as unaligned but could just be my weird interface. My point was to delete the blank lines :D

Reply:
I'll delete the blank lines.